foundation/application.go

```diff
@@ -129,7 +129,7 @@ func (app *Application) Version() string {
 func (app *Application) CurrentLocale(ctx context.Context) string {
 	lang := app.MakeLang(ctx)
 	if lang == nil {
-		color.Red().Println("Error: Lang facade not initialized.")
+		color.Errorln("Error: Lang facade not initialized.")
```

Comment:
we can add a errors variable in `errors/list.go` and `Errorln` will automatically add `Error` prefix.

Reply:
Good catch, actually, `color.Errorln` has the Error prefix, `Error:` can be removed directly.